### PR TITLE
Mention IN2P3 gitlab set of tutorials.

### PR DESCRIPTION
**Pull request by chavid**

### CODE REVIEW

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 1]
```diff
@@ -562,14 +562,24 @@ \subsection{Initiatives for future training programs\label{sec:initiatives}}
...
+the learning curve has been kept appropriately shallow. An interesting exercise in
+this context has recently been made by colleagues of
 the GridKa School of computing: the material from this year (2017) has been made
 publicly available on WTL\footnote{\url{https://en.wikitolearn.org/GridKa2017}}.
-This constitutes an intersting example of what can be accomplished using this
+This constitutes an interesting example of what can be accomplished using this
 platform; it is just a first example of what's possible, but an inspiring one.
 
+Another example of web-based platform is a collection of online tutorials
+(mostly written in french) hosted on the Gitlab IN2P3
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: french -> French

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 2]
```diff
@@ -562,14 +562,24 @@ \subsection{Initiatives for future training programs\label{sec:initiatives}}
...
+server{\url{https://gitlab.in2p3.fr/MaitresNageurs}}. Taking profit of
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: Make this a citation, adding the reference and link to the cwp.bib file.

Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: Taking profit of -> Taking advantage of

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 3]
```diff
@@ -562,14 +562,24 @@ \subsection{Initiatives for future training programs\label{sec:initiatives}}
...
+the Gitlab ability to host Docker images, those tutorials aims to avoid the
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: aims -> aim

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 4]
```diff
@@ -562,14 +562,24 @@ \subsection{Initiatives for future training programs\label{sec:initiatives}}
...
+"damned installation step" that often absorbs half of a training session.
+Similarly to the GridKa School for WikiToLearn, the annual IN2P3 Computing
+Days are an opportunity to refresh and extend the collection of
+tutorials every year. Future work will focus on english translation, and the
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: english -> English

**Comment by chavid** [comment]
Changes made.

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 5]
```diff
@@ -480,13 +480,23 @@ \subsection{Initiatives for future training programmes\label{sec:initiatives}}
...
 publicly available on WTL~\cite{WikiToLearnGridka}.
-This constitutes an intersting example of what can be accomplished using this
+This constitutes an interesting example of what can be accomplished using this
 platform; it is just a first example of what's possible, but an inspiring one.
 
+Another example of web-based platform is a collection of online tutorials
+(mostly written in French) hosted on the Gitlab IN2P3
+server~\cite{Code-Swim-Coaches}. Taking advantage of
+the Gitlab ability to host Docker images, those tutorials aim to avoid the
+"damned installation step" that often absorbs half of a training session.
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: Need to fix quotes here for LaTeX, ``damned installation step''.

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex [hunk 6]
```diff
@@ -480,13 +480,23 @@ \subsection{Initiatives for future training programmes\label{sec:initiatives}}
...
+Similarly to the GridKa School for WikiToLearn, the annual IN2P3 Computing
```
Comment (graeme-a-stewart) [inline CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: Add citations - I think they are in the `cwp.bib` file already.

Comment (chavid) [reply CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: You mean repeat the citations for GridKa and WikiToLearn which are already made in the paragraphs just above ?

File: CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex (hunk 5 above)
Comment (chavid) [reply CWP/papers/HSF-CWP-2017-02_training/latex/cwp-wg-training-careers.tex]: Done.

**Comment by chavid** [comment]
Changes made.

**Pull request merged**